apiCalls.js

```diff
@@ -0,0 +1,37 @@
+const fetch = require("node-fetch");
+
+const movieFetch = async (table) => {
+  console.log(table)
+  try {
+    const response = await fetch(`https://imdb8.p.rapidapi.com/title/get-popular-movies-by-genre?genre=/chart/popular/genre/${table}`, {
+      "method": "GET",
+      "headers": {
+        "x-rapidapi-host": "imdb8.p.rapidapi.com",
+        "x-rapidapi-key": "580db340e1mshd3d03efa046e426p182a53jsnf5a8138fcc1a"
+      }
+    })
+    const resolved = await response.json()
+    return resolved
+  }
+  catch (e) {
+    console.log(e);
+  }
+}
+const singleMovie = async (title) => {
+  try {
+    const response = await fetch(`https://imdb8.p.rapidapi.com/title/get-overview-details?currentCountry=US&tconst=${title}`, {
+      "method": "GET",
+      "headers": {
+        "x-rapidapi-host": "imdb8.p.rapidapi.com",
+        "x-rapidapi-key": "580db340e1mshd3d03efa046e426p182a53jsnf5a8138fcc1a"
+      }
+    })
+    const resolved = await response.json()
+    return resolved
+  }
+  catch (e) {
```

Comment:
Is anything supposed to happen inside of the `catch`?

Reply:
I think it should only return the `error`, but this is now making me wonder what would happen if we ever get a bad response at this point. Do we just need to console log the error?